go.mod

```diff
@@ -75,7 +68,10 @@ replace (
 	github.com/heroku/docker-registry-client => github.com/stackrox/docker-registry-client v0.0.0-20220204234128-07f109db0819
 	github.com/mattn/goveralls => github.com/viswajithiii/goveralls v0.0.3-0.20190917224517-4dd02c532775
 	github.com/nilslice/protolock => github.com/viswajithiii/protolock v0.10.1-0.20190117180626-43bb8a9ba4e8
-	github.com/stackrox/rox => github.com/stackrox/stackrox v0.0.0-20210914215712-04ef1c1cfc62
+
+	github.com/operator-framework/helm-operator-plugins => github.com/stackrox/helm-operator v0.0.8-0.20220506091602-3764c49abfb3
+	github.com/stackrox/rox => github.com/stackrox/stackrox v0.0.0-20220512161225-64d3c8bd40a8
```

Comment:
This was updated to bring in `clientconn.NewHTTPClient()` from `stackrox/stackrox`, but I am now 100% confident that's the best way to do it, and my knowledge of go's module dependency is very limited.